src/components/Post.js

```diff
@@ -0,0 +1,27 @@
+import { Avatar, Card, Typography } from "@material-ui/core";
+import React from "react";
+
+export const Post = ({ authorImage, authorName, timestamp, children }) => (
+  <Card
+    style={{
+      maxWidth: "600px",
+      padding: "10px 10px",
+      marginBottom: "10px",
+      display: "flex",
+      alignItems: "center",
+      flexDirection: "row",
+      justifyContent: "space-around",
+    }}
+  >
+    <Avatar src={authorImage} />
+    <div>
+      <Typography variant="h6">{children}</Typography>
+      <Typography>
+        Gepostet von: {authorName}{" "}
+        {typeof timestamp === "string"
+          ? "um " + timestamp
+          : "am " + timestamp.toDate().toLocaleString("de-DE")}
```

Comment:
This is good. I was wondering how to do this.

Reply:
Ideally you could manually move every current string time into a firebase timestamp from the console to remove the condition